app/_src/policies/introduction.md

```diff
@@ -1,6 +1,661 @@
+- If levels are equal the lexicographic order of policy names is used
+
+{% tip %}
+Remember that a broader a targetRef is the lower its priority is.
+{% endtip %}
+
+For `to` and `from` policies we concatenate the array for each matching policies.
+We then build configuration by merging each level using [JSON patch merge](https://www.rfc-editor.org/rfc/rfc7386).
+
+For example if I have 2 `default` ordered this way:
+
+```yaml
+default:
+  conf: 1
+  sub:
+    array: [1, 2, 3]
+    other: 50
+    other-array: [3, 4, 5]
+---
+default:
+  sub:
+    array: []
+    other: null
+    other-array: [5, 6]
+    extra: 2
+```
+
+The merge result is:
+
+```yaml
+default:
+  conf: 1
+  sub:
+    array: []
+    other-array: [5, 6]
+    extra: 2
+```
+
+## Using policies with `MeshService`, `MeshMultizoneService` and `MeshExternalService`.
+
+[`MeshService`](/docs/{{ page.version }}/networking/meshservice) is a feature to define services explicitly in {{ site.product_name }}.
+It can be selectively enabled and disable depending on the value of [meshServices.mode](/docs/{{ page.version }}/networking/meshservice/#migration) on your Mesh object.
+
+When using explicit services, `MeshServiceSubset` is no longer a valid kind and `MeshService` can only be used to select an actual `MeshService` resource (it can no longer select a `kuma.io/service`).
+
+In the following example we'll assume we have a `MeshService`:
+
+{% policy_yaml ms-1 namespace=kuma-demo %}
+```yaml
+type: MeshService
+name: my-service
+labels:
+  k8s.kuma.io/namespace: kuma-demo 
+  kuma.io/zone: my-zone
+  app: redis 
+spec:
+  selector:
+    dataplaneTags:
+      app: redis
+      k8s.kuma.io/namespace: kuma-demo 
+  ports:
+  - port: 6739
+    targetPort: 6739
+    appProtocol: tcp
+```
+{% endpolicy_yaml %}
+
+There are 2 ways to select a `MeshService`:
+
+If you are in the same namespace (or same zone in Universal) you can select one specific service by using its explicit name:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: timeout-to-redis
+  namespace: kuma-demo
+spec:
+  to:
+  - targetRef:
+      kind: MeshService
+      name: redis
+    default:
+      connectionTimeout: 10s
+```
+
+Selecting all matching `MeshServices` by labels:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: all-in-my-namespace 
+  namespace: kuma-demo
+spec:
+  to:
+  - targetRef:
+      kind: MeshService
+      labels:
+        k8s.kuma.io/namespace: kuma-demo
+    default:
+      connectionTimeout: 10s
+```
+
+In this case this is equivalent to writing a specific policy for each service that matches this label (in our example for each service in this namespace in each zones).
+
+{% tip %}
+When `MeshService` have multiple ports, you can use `sectionName` to restrict policy to a single port.
+{% endtip %}
+
+### Global, Zonal, Producer and Consumer policies
```

Comment:
Inconsistent capitalization. I _think_ we agreed on only the first word. Either way this isn't right.